admin/Dockerfile.qa

```diff
@@ -18,17 +18,20 @@ ARG CXX
 # Solution: install python3-setuptools (includes Python 3.6),
 # then use the get-pip script from Python 3.7.
 RUN \
-    apt-get update && \
-    apt-get install -y sudo \
-                       make \
-                       $CC \
-                       $CXX \
-                       python3-setuptools \
-                       $(if [ "$DOCKER_OS" != "ubuntu:22.04" ]; then echo python3.7; fi) \
-                       ninja-build \
-                       curl \
-                       libxml2-utils && \
-    apt-get clean
+    export DEBIAN_FRONTEND=noninteractive&& apt-get update \
```

Comment:
Does not follow the same pattern as the follow on commands. It would be cleaner if: 
export DEBIAN_FRONTEND=noninteractive \
&& apt-get update \